src/server/req_jobobit.c

```diff
@@ -1490,6 +1490,9 @@ on_job_rerun(struct work_task *ptask)
 
 				}
 				rel_resc(pjob);		/* free resc assigned to job */
```

Comment:
only one minor suggestion - 
add a comment here saying "respond to pending preemption request from the scheduler, if any"